### PR TITLE
source-zendesk-sell: fix phone_number type from number to string #29828

### PR DESCRIPTION
**Pull request by speckm**
## What
This patch updates the definition of the calls entity in source-zendesk

## How
The patch updates the calls.json file of source-zendesk-sell and changes the type from number to string

## 🚨 User Impact 🚨
This change could impact users, where the target relies on the type number.

### CODE REVIEW

File: airbyte-integrations/connectors/source-zendesk-sell/source_zendesk_sell/schemas/calls.json [hunk 1]
```diff
@@ -21,7 +21,7 @@
       "type": ["null", "number"]
     },
     "phone_number": {
-      "type": ["null", "number"]
+      "type": ["null", "string"]
```
Comment (marcosmarxm) [inline airbyte-integrations/connectors/source-zendesk-sell/source_zendesk_sell/schemas/calls.json:24]: The API says `phone_number` is string https://developer.zendesk.com/api-reference/sales-crm/resources/calls/ Do you have another info @speckm ?

**Review by marcosmarxm — CHANGES_REQUESTED** [review]
@speckm please sign the CLA.

File: airbyte-integrations/connectors/source-zendesk-sell/source_zendesk_sell/schemas/calls.json (hunk 1 above)
Comment (speckm) [reply airbyte-integrations/connectors/source-zendesk-sell/source_zendesk_sell/schemas/calls.json:24]: The API says `number` not `string`.  This seems to be a documentation error. The data which you get from a request includes strings. Here is a excerpt from a call to the api (made 2023-08-27):

```
{
    id: 134328917,
    user_id: 6907886,
    summary: 'Energieausweis auf H, 3 WE, Haus frisch gekauft',
    recording_url: null,
    outcome_id: null,
    duration: null,
    phone_number: '+491762203????',
    incoming: false,
    missed: false,
    updated_at: '2023-08-18T08:46:33.000Z',
    external_id: null,
    associated_deal_ids: [ 159638??? ],
    made_at: '2023-08-18T08:46:33Z',
    resource_id: 473697140,
    resource_type: 'contact'
  },
  {
    id: 134328735,
    user_id: 6907886,
    summary: null,
    recording_url: null,
    outcome_id: 4974167,
    duration: null,
    phone_number: '+49171321????',
    incoming: false,
    missed: false,
    updated_at: '2023-08-18T08:40:49.000Z',
    external_id: null,
    associated_deal_ids: [ 159420??? ],
    made_at: '2023-08-18T08:40:49Z',
    resource_id: 473105065,
    resource_type: 'contact'
  },
```

I switched some number to ? to keep data save.

**Review by marcosmarxm — APPROVED** [review]

File: airbyte-integrations/connectors/source-zendesk-sell/metadata.yaml [hunk 2]
```diff
@@ -2,7 +2,7 @@ data:
   connectorSubtype: api
   connectorType: source
   definitionId: 982eaa4c-bba1-4cce-a971-06a41f700b8c
-  dockerImageTag: 0.1.0
+  dockerImageTag: 0.1.1
```
Comment (marcosmarxm) [inline airbyte-integrations/connectors/source-zendesk-sell/metadata.yaml:5]: @speckm can you bump the connector doc changelog entry? After we can merge the contribution!

**Review by marcosmarxm — CHANGES_REQUESTED** [review]

File: airbyte-integrations/connectors/source-zendesk-sell/metadata.yaml (hunk 2 above)
Comment (speckm) [reply airbyte-integrations/connectors/source-zendesk-sell/metadata.yaml:5]: I added a changelog entry für version 0.1.1.

**Comment by octavia-approvington** [comment]
Our work here is done <br/> done

**Review by octavia-approvington — APPROVED** [review]

**Pull request merged**